### ISSUE TITLE
The camelCase option needs to support string values

### ISSUE THREAD

**Issue by evisong** [michalkvasnicak/babel-plugin-css-modules-transform#40]
Hi, 

Thanks for this lib, we're heavily using it in our project.

Regarding the `camelCase` option, setting it to `true` causes a lot of duplications since original class names are kept, e.g.:

```js
var Styles = {
  'link': 'link-modules__link___3oOIo',
  'linkInGroup': 'link-modules__link-in-group___2fh2a',
  'linkInLinks': 'link-modules__link-in-links___3IT7k',
  'linkDisabled': 'link-modules__link-disabled___2qbP8',
  'link-in-group': 'link-modules__link-in-group___2fh2a',
  'link-in-links': 'link-modules__link-in-links___3IT7k',
  'link-disabled': 'link-modules__link-disabled___2qbP8'
};
```

In order to solve this, `css-modules-require-hook` has just supported new string values: `only`|`dashOnly` per https://github.com/css-modules/css-modules-require-hook/issues/89 , and released in v4.0.6 https://github.com/css-modules/css-modules-require-hook/releases/tag/4.0.6

> camelCase: true|'dashes'|'dashesOnly'|'only'

In latest `babel-plugin-css-modules-transform`, I found that it limits `camelCase` to be a boolean: 

```
Error: Configuration 'camelCase' is not a boolean
    at Object.camelCase (.../node_modules/babel-plugin-css-modules-transform/build/options_resolvers/camelCase.js:18:15)
```

Is it possible to support the new values ('dashes'|'dashesOnly'|'only')?

Thanks.

**Comment by michalkvasnicak** [comment]
Thank you! I will update dependencies and modify [camelCase](https://github.com/michalkvasnicak/babel-plugin-css-modules-transform/blob/master/src/options_resolvers/camelCase.js) option resolver.

**Comment by michalkvasnicak** [comment]
Released as `1.2.7`

**Comment by evisong** [comment]
I'm totally impressed by your super fast response, thanks a lot!
